commons-dlp/src/main/java/org/exoplatform/commons/dlp/service/impl/DlpPositiveItemServiceImpl.java

```diff
@@ -4,75 +4,83 @@
 import org.exoplatform.commons.dlp.domain.DlpPositiveItemEntity;
 import org.exoplatform.commons.dlp.dto.DlpPositiveItem;
 import org.exoplatform.commons.dlp.service.DlpPositiveItemService;
+import org.exoplatform.services.listener.Event;
+import org.exoplatform.services.listener.ListenerService;
 import org.exoplatform.services.log.ExoLogger;
 import org.exoplatform.services.log.Log;
-import org.exoplatform.services.organization.OrganizationService;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class DlpPositiveItemServiceImpl implements DlpPositiveItemService {
 
-    private final DlpPositiveItemDAO dlpPositiveItemDAO;
+  private static final Log LOG =
+      ExoLogger.getLogger(DlpPositiveItemServiceImpl.class);
 
-    private static final Log LOG =
-            ExoLogger.getLogger(DlpPositiveItemServiceImpl.class);
+  private final DlpPositiveItemDAO dlpPositiveItemDAO;
 
-    public DlpPositiveItemServiceImpl(DlpPositiveItemDAO dlpPositiveItemDAO) {
-        this.dlpPositiveItemDAO = dlpPositiveItemDAO;
-    }
+  private ListenerService listenerService;
 
+  public DlpPositiveItemServiceImpl(DlpPositiveItemDAO dlpPositiveItemDAO, ListenerService listenerService) {
+    this.dlpPositiveItemDAO = dlpPositiveItemDAO;
+    this.listenerService = listenerService;
+  }
 
-    @Override
-    public List<DlpPositiveItem> getDlpPositivesItems(int offset, int limit) throws Exception {
-        List<DlpPositiveItemEntity> dlpPositiveItemEntities = dlpPositiveItemDAO.getDlpPositiveItems(offset, limit);
-        return fillDlpPositiveItemsFromEntities(dlpPositiveItemEntities);
-    }
+  @Override
+  public List<DlpPositiveItem> getDlpPositivesItems(int offset, int limit) throws Exception {
+    List<DlpPositiveItemEntity> dlpPositiveItemEntities = dlpPositiveItemDAO.getDlpPositiveItems(offset, limit);
+    return fillDlpPositiveItemsFromEntities(dlpPositiveItemEntities);
+  }
 
-    @Override
-    public void addDlpPositiveItem(DlpPositiveItemEntity dlpPositiveItemEntity) {
-        dlpPositiveItemDAO.create(dlpPositiveItemEntity);
-    }
+  @Override
+  public void addDlpPositiveItem(DlpPositiveItemEntity dlpPositiveItemEntity) {
+    dlpPositiveItemDAO.create(dlpPositiveItemEntity);
+  }
 
-    @Override
-    public void deleteDlpPositiveItem(Long itemId) {
-        DlpPositiveItemEntity dlpPositiveItemEntity = dlpPositiveItemDAO.find(itemId);
-        if (dlpPositiveItemEntity != null) {
-            dlpPositiveItemDAO.delete(dlpPositiveItemEntity);
-        } else {
-            LOG.warn("The DlpItem's {} not found.", itemId);
-        }
+  @Override
+  public void deleteDlpPositiveItem(Long itemId) {
```

Comment:
Can you add a log on this ? 
Example : https://github.com/exoplatform/ecms/blob/7f46e53e87f3c2707d54c580b3188da86370afd8/core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java#L125
So that we can see when a DLPPositiveItem is deleted
Thanks